geopandas/tools/crs.py

```diff
@@ -0,0 +1,38 @@
+import pyproj
+import re
+import os
+import fiona.crs
+
+
+def explicit_crs_from_epsg(crs=None, epsg=None):
+    """
+    Gets full/explicit CRS from EPSG code provided.
+
+    Parameters
+    ----------
+     crs : dict or string, default None
+         An existing crs dict or string with the 'init' key specifying an EPSG code
+     epsg : string or int, default None
+        The EPSG code to lookup
+    """
+
+    if epsg is None:
+        if crs is not None:
+            if isinstance(crs, str):
+                crs = fiona.crs.from_string(crs)
+            if 'init' in crs and crs['init'].lower().startswith('epsg:'):
+                epsg = crs['init'].split(':')[1]
+
+    if epsg is None:
+        raise ValueError('No epsg code provided.')
+
+    with open(os.path.join(pyproj.pyproj_datadir, 'epsg')) as f:
+        data = f.read()
+
+    _crs = re.search('\n<{}>\s*(.+?)\s*<>'.format(epsg), data)
+    if _crs is None:
+        raise ValueError('EPSG code "{}" not found.'.format(epsg))
```

Comment:
Does this ever fail for valid EPSG codes? I can certainly imagine that the answer is "no", but curious with respect to error handling.

Reply:
Any epsg code which is usable in pyproj will be found using this regex. The file is a consistent format so this regex should function reliably.